environment.yml

```diff
@@ -26,6 +26,7 @@ dependencies:
   - networkx
   - beautifulsoup4
   - pip
+  - compilers
   - pip:
       - git+https://github.com/openforcefield/Arsenic.git@0.1.0
```

Comment:
There's now an `arsenic` conda-forge package!
https://anaconda.org/conda-forge/arsenic

Reply:
Ah perfect! We'll switch to that.